src/CxbxKrnl/HLEDataBase/XactEng.1.0.5849.inl

```diff
@@ -57,11 +57,32 @@ OOVPA_NO_XREF(XACTEngineCreate, 5849, 11)
 OOVPA_END;
 
 // ******************************************************************
+// * XACT::CEngine::CreateSoundSource
+// ******************************************************************
+// Created from $0030420C in "Kingdom Under Fire: The Crusaders"
+OOVPA_XREF(XACT_CEngine_CreateSoundSource, 5849, 8,
+		XREF_XACT_CEngine_CreateSoundSource,
+		XRefZero)
+		//XACT::CEngine::CreateSoundSource+0x09 : push eax
+		{ 0x09, 0x50 },
+		//XACT::CEngine::CreateSoundSource+0x0F : or eax, 20000000h
```

Comment:
Could you perhaps still use the or opcode, without the 20000000h constant, and instead add some additional opcode or register operations?

It's best to avoid constants just-in-case they change per title, or in a later XDK (often, OOVPA can be reused)

Reply:
Sure, no problem - I'll update it tonight. I didn't think about that.